compiler/AST/view.cpp

```diff
@@ -26,6 +26,7 @@
 #include "AstDump.h"
```

Comment:
Ah, I have my vim config set up to clean up trailing whitespace in files that I touch. Since I needed to add the include, it cleaned up the whitespace as well.

Is that alright or would you still prefer that I revert?